app/src/main/java/com/safetyheads/akademiaandroida/SplashScreenActivity.kt

```diff
@@ -0,0 +1,21 @@
+package com.safetyheads.akademiaandroida
+
+
+import android.content.Intent
+import android.os.Bundle
+import android.os.Handler
+import androidx.appcompat.app.AppCompatActivity
+
+
+class SplashScreenActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_splash_screen)
+
+        Handler().postDelayed({
```

Comment:
Please handle case when user put app into background, activity shouldn't started. Timer should be restarted.